angr/analyses/cfg/cfg_fast.py

```diff
@@ -789,10 +789,8 @@ class CFGFast(ForwardAnalysis, CFGBase):    # pylint: disable=abstract-method
 
     # Active scanning
 
-        - If the binary has "function symbols" (TODO: this term is not accurate enough), they are starting points of
-            the code scanning
-        - If the binary does not have any "function symbol", we will first perform a function prologue scanning on the
-            entire binary, and start from those places that look like function beginnings
+        - If the binary has "function symbols" (TODO: this term is not accurate enough), they are starting points of the code scanning
+        - If the binary does not have any "function symbol", we will first perform a function prologue scanning on the entire binary, and start from those places that look like function beginnings
```

Comment:
We want to keep our lines at 100 characters or less. You can do line extensions in RST by having the line continuation be at the same indentation level as the start of the text on the previous line, if I recall correctly. RST is tricky...